geoviews/plotting/mpl/chart.py

```diff
@@ -0,0 +1,156 @@
+import param
+import numpy as np
+from holoviews.plotting.mpl.element import ColorbarPlot
+from holoviews.util.transform import dim
+from holoviews.core.options import abbreviated_exception
+
+
+class WindBarbsPlot(ColorbarPlot):
+    """
+    Barbs are traditionally used in meteorology as a way to plot the speed and
+    direction of wind observations, but can technically be used to plot any two
+    dimensional vector quantity. As opposed to arrows, which give vector
+    magnitude by the length of the arrow, the barbs give more quantitative
+    information about the vector magnitude by putting slanted lines or a
+    triangle for various increments in magnitude.
+
+    The largest increment is given by a triangle (or "flag"). After those come full lines (barbs).
+    The smallest increment is a half line. There is only, of course, ever at most 1 half line.
+    If the magnitude is small and only needs a single half-line and no full lines or
+    triangles, the half-line is offset from the end of the barb so that it can be
+    easily distinguished from barbs with a single full line. The magnitude for the barb
+    shown above would nominally be 65, using the standard increments of 50, 10, and 5.
+    """
+
+    padding = param.ClassSelector(default=0.05, class_=(int, float, tuple))
+
+    from_uv_components = param.Boolean(default=False, doc="""
```

Comment:
I would add the data conversion to the `geoviews.element.geo.WindBarbs`. So it is possible in the future to use it for other backends.

My preference would be to have this functionality as a `classmethod`. So you could do something like WindBarbs.from_uv_componts(u, v)`.

Reply:
Or just `from_uv`, as `uv` is a pretty strong indicator of `components`.

Reply:
Added `from_uv` classmethod.